org.mixedrealitytoolkit.input/InteractionModes/IXRProximityInteractable.cs

```diff
@@ -18,15 +18,13 @@ public interface IXRProximityInteractable
         /// <summary>
         /// Registers the duple Collider + XRBaseInteractor as triggering proximity.
         /// </summary>
-        /// <param name="collider">Collider triggering proximity.</param>
-        /// <param name="xrBaseInteractor">Interactor triggering proximity.</param>
-        void OnProximityEntered(ProximityEnteredEventArgs args);
+        /// <param name="args">ProximityHoverEnteredEventArgs that has the Collider + Interactable duple triggering the proximity-hover event.</param>
```

Comment:
Is this comment still true? I don't think it has the `Collider` anymore

Reply:
Nope, not anymore.  I missed that when updating comments in a previous commit.  Thanks for catching that and letting me know.  Fixed in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/659/commits/29a7dee6124468197e8ac62359687d173eeb50c4